pages/index.js

```diff
@@ -8,13 +11,28 @@ export default function Home() {
 	const Form = dynamic(() => import('../src/components/Form/Form'), {
 		ssr: false, // This line is important.
 	});
+
+	const archiveTasks = useStore(state => state.archiveTasks);
+
 	return (
 		<main>
 			<h1 hidden>ToNotDo</h1>
-			<h2>Create</h2>
+			<h2 hidden>Create</h2>
 			<Form editMode={false} />
-			<h2>ToDo</h2>
+			<h2 hidden>ToDo</h2>
 			<TaskList />
+			<article>
+				<Graph />
+			</article>
+			<article>
+				<Button
+					onClick={() => {
+						archiveTasks();
+					}}
+				>
+					Archive checked
+				</Button>
+			</article>
```

Comment:
why do you use the article tags here?

Reply:
I styled my listelements by an article and wanted the button to be nested in the same style. I should probably use a className instead. I will check that in the next pull request.